lib/view/widget/button/pictron_base_button.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:flutter/material.dart';
+
+class PictronBaseButton extends ButtonBar {
+  PictronBaseButton({
+    Key key,
+    String text = "",
+    Function function,
+    TextStyle textStyle,
+  }) : super(
+          children: <Widget>[
+            RaisedButton(
+                onPressed: () {
+                  function();
+                },
+                child: Text(text, style: textStyle))
```

Comment:
It's a bit tricky to have two closing parentheses characters here. The last one closes the RaisedButton element, so it should be at a new line.